building/get-apt-signing-key.sh

```diff
@@ -0,0 +1,21 @@
+while read branch key; do
+	if [ "${branch}" == "${HOMEWORLD_APT_BRANCH}" ] || [ "${branch}" == "$(echo ${HOMEWORLD_APT_BRANCH} | cut -d '/' -f 1)" ]; then
+		HOMEWORLD_APT_SIGNING_KEY="${key}"
+		break
+	fi
+done < signing-keys
```

Comment:
For the sake of readability, I think people usually wrap constructs in parentheses if they're going to have things redirected into them afterwards?

i.e.:

    (for x in y; do
        <code>
    done) > output-file